flink-examples/flink-examples-table/src/main/scala/org/apache/flink/table/examples/scala/StreamSQLExample.scala

```diff
@@ -49,12 +49,16 @@ object StreamSQLExample {
     val env = StreamExecutionEnvironment.getExecutionEnvironment
     val tEnv = if (planner == "blink") {  // use blink planner in streaming mode
```

Comment:
Minor: `EnvironmentSettings.Builder settingsBuilder = EnvironmentSettings.newInstance().inStreamingMode();` to reuse `StreamTableEnvironment.create` code.
`StreamSQLExample.java` too.

Reply:
I think current is also fine.